README.md

```diff
@@ -5,5 +5,35 @@ This is a sample of a Java Spring app that works with Tilt and the Tanzu Applica
 ## Dependencies
 1. [kubectl CLI](https://kubernetes.io/docs/tasks/tools/)
 1. [Tilt version >= v0.22.6](https://docs.tilt.dev/install.html)
-1. [Tanzu CLI apps plugin v0.2.0](https://github.com/vmware-tanzu/tanzu-cli-apps-plugins)
-1. A cluster with Tanzu Application Platform, the Inner Loop Supply Chain, and their dependencies. Download these from Tanzunet.
+1. Tanzu CLI and the apps plugin v0.2.0 which are provided as part of [Tanzu Application Platform](https://network.tanzu.vmware.com/products/tanzu-application-platform)
+1. A cluster with Tanzu Application Platform, and one of the provided supply chains, plus its dependencies. These supply chains are part of [Tanzu Application Platform](https://network.tanzu.vmware.com/products/tanzu-application-platform).
+
+## Running the sample
+
+Start the app deployment by running:
+
+```
+tilt up
+```
+
+You can hit the spacebar to open the UI in a browser. 
+
+- > If you see an "Update error" message like the one below, then just follow the instructions and allow that context:
+    ```
+    Stop! tap-beta2 might be production.
+    If you're sure you want to deploy there, add:
+        allow_k8s_contexts('tap-beta2')
+    to your Tiltfile. Otherwise, switch k8s contexts and restart Tilt.
+    ```
+
+Once the workload is up and running you can access the app on the local port that Tilt is forwarding:
+
+```
+curl -w'\n' localhost:8080
```

Comment:
Sorry I didn't catch this in last review - the Tiltfile in this repo does not configure port-forwarding, so the way to access the app is through its knative service url.

Reply:
cool glad you decoupled this to allow the rest to move forward. Feel free to open a separate issue for the curl thing. I'm not sure why it worked for you.